commons/src/main/java/org/infinispan/lock/api/ClusteredLockConfiguration.java

```diff
@@ -0,0 +1,41 @@
+package org.infinispan.lock.api;
+
+import org.infinispan.commons.util.Experimental;
+
+/**
+ * A Clustered Lock can be rentrant and there are different ownership levels.
+ * <p>
+ * The only mode supported now is "non rentrant" locks for "nodes".
+ *
+ * @author Katia Aresti, karesti@redhat.com
+ * @see <a href="http://infinispan.org/documentation/">Infinispan documentation</a>
+ * @since 9.2
+ */
+@Experimental
+public class ClusteredLockConfiguration {
+   private final OwnershipLevel ownershipLevel; // default NODE
+   private final boolean rentrant; // default false
+
+   /**
+    * Default lock is non rentrant and the ownershop level is {@link OwnershipLevel#NODE}
+    */
+   public ClusteredLockConfiguration() {
+      this.ownershipLevel = OwnershipLevel.NODE;
+      this.rentrant = false;
+   }
+
+   /**
+    * @return true if the lock is rentrant
```

Comment:
typo `rentrant` => `reentrant`
ps. maybe a find a replace will do the trick so I don't have to point them all :)

Reply:
ok